osu.Game/Graphics/ScreenshotManager.cs

```diff
@@ -62,6 +68,12 @@ private void load(OsuConfigManager config, Storage storage, AudioManager audio)
             captureMenuCursor = config.GetBindable<bool>(OsuSetting.ScreenshotCaptureMenuCursor);
 
             shutter = audio.Samples.Get("UI/shutter");
+
+            posX = config.GetBindable<float>(OsuSetting.ScalingPositionX);
```

Comment:
Rather than getting bindables for these, it's probably better to just request them inline. I've made this change.